cfgov/ask_cfpb/fixtures/portal_categories.json

```diff
@@ -4,79 +4,39 @@
     "pk": 1,
     "fields": {
         "heading": "Basics",
-        "heading_es": ""
+        "heading_es": "Lo b\u00e1sico"
```

Comment:
Do we need these updates, or this fixture more generally, now that they've been initialized in the Wagtail admin?

Reply:
They're useful for testing.